account/src/use-case/createUserAccount.js

```diff
@@ -1,14 +1,17 @@
 import crypto from 'crypto'
 import { saveAccount } from "../../repositories/accountRepository.js";
+import bcrypt from 'bcryptjs'
+
 
 export async function createUserUseCase(name, email, password) {
   const createdDate = new Date().toISOString().substring(0, 10);
+  const hashedPassword = bcrypt.hashSync(password, parseInt(process.env.HASH_SALT));
```

Comment:
i think the right approach is to create a variable with the result before and put the variable here after